site/cypress/e2e/cypress-feature/grading_stats.spec.js

```diff
@@ -1,160 +1,68 @@
+const ApplyFilter = () => {
+    cy.get('#filters').click();
+    cy.get('#bad_submissions').click();
+    cy.get('#apply_button').click();
+};
+
 describe('Test cases for grading stats', () => {
+
     ['instructor','ta','grader'].forEach((user) => {
         beforeEach(() => {
             cy.login(user);
             cy.visit(['sample']);
```

Comment:
This line is now useless, because of the change from clicking buttons to visiting the final page.